tarpc/src/serde_transport.rs

```diff
@@ -433,22 +433,27 @@ pub mod unix {
     pub struct TempSock(std::path::PathBuf);
```

Comment:
Nit, thinking about the naming: this is actually not creating the socket; it's choosing the place to create it, and then it's making sure it's cleaned up later. But it could be used for non-socket purposes, too (not that we intend it to be...).

What about something like FileCleanup or RemoveFileOnDrop?

Reply:
Great point! Perhaps in this case sticking close to what it actually is would be a benefit? So something like `TempPathBuf` and just remove the `.sock` stuff. I'll do that real quick and add the tests for the naming as well.

Reply:
Changed the name and removed `.sock` part of the names in: [fd0e243](https://github.com/google/tarpc/pull/380/commits/fd0e2434c0dfd7f1b276ef07ed64d77c8c942a13)